Tests/Unit/NetworkServer/JsonHandlers/LnsStationConfigurationTests.cs

```diff
@@ -580,5 +580,110 @@ public void RegionConfigurationConverter_Throws_OnNotSetRegion()
+                        'tx_freq_min': 920000000,
+                        'th_freq_max': 923400000
+                    }},
+                    'radio_1': {{
+                        'enable': true,
+                        'type': 'SX1257',
+                        'freq': {1},
+                        'rssi_offset': -166.0,
+                        'tx_enable': false,
+                    }},
+                    'chan_FSK': {{
+                        'enable': true,
+                        'radio': 1,
+                        'if': -1800000
+                    }},
+                    'chan_Lora_std': {{
+                        'enable': true,
+                        'radio': 1,
+                        'if': -1800000,
+                        'bandwidth': 250000,
+                        'spread_factor': 7
+                    }},
+                    'chan_multiSF_0': {{
+                        'enable': true,
+                        'radio': 0,
+                        'if': {2}
+                    }},
+                    'chan_multiSF_1': {{
+                        'enable': true,
+                        'radio': 0,
+                        'if': {3}
+                    }},
+                    'chan_multiSF_2': {{
+                        'enable': true,
+                        'radio': 1,
+                        'if': 0
+                    }},
+                    'chan_multiSF_3': {{
+                        'enable': true,
+                        'radio': 0,
+                        'if': -1800000
+                    }},
+                    'chan_multiSF_4': {{
+                        'enable': true,
+                        'radio': 0,
+                        'if': -1800000
+                    }},
+                    'chan_multiSF_5': {{
+                        'enable': true,
+                        'radio': 0,
+                        'if': 0
+                    }},
+                    'chan_multiSF_6': {{
+                        'enable': true,
+                        'radio': 0,
+                        'if': -1800000
+                    }},
+                    'chan_multiSF_7': {{
+                        'enable': true,
+                        'radio': 0,
+                        'if': -1800000
+                    }}
+                }}
+            ],
+            'nocca': true,
+            'nodc': true,
+            'nodwell': true}}";
+
+            var config1 = JsonUtil.Strictify(string.Format(CultureInfo.InvariantCulture, input, 923000000, 922000000, 200000, 400000));
+            var region1 = LnsStationConfiguration.GetRegion(config1);
+
+            var config2 = JsonUtil.Strictify(string.Format(CultureInfo.InvariantCulture, input, 923000000, 922000000, -1600000, -1400000));
+            var region2 = LnsStationConfiguration.GetRegion(config2);
+
+            // asserting that reading region2 is not affecting region1
+            Assert.Equal(typeof(RegionAS923), region1.GetType());
```

Comment:
Consider using `Assert.IsType` and leveraging the return value to continue with further assertions (without needing to cast)

Reply:
Done